handlers/overview.go

```diff
@@ -368,8 +369,10 @@ func filterFlexOverview(w http.ResponseWriter, req *http.Request, f *FilterFlex,
 	}
 
 	basePage := f.Render.NewBasePageModel()
+	custom := helpers.IsBoolPtr(filterJob.Custom)
```

Comment:
Is this needed here?

Reply:
It was for some reason - not sure why but I had to wade an ocean of logging to figure it